src/shared/orgShapeListUtils.ts

```diff
@@ -16,22 +16,25 @@ interface OrgShape {
   CreatedDate: string;
 }
 
-export interface OrgShapeListResult {
+export type OrgShapeListResult = {
   orgId: string;
   username: string;
   alias?: string;
   shapeId: string;
   status: string;
   createdBy: string;
   createdDate: string;
-}
+};
 
 export interface JsForceError extends Error {
   errorCode: string;
   fields: string[];
 }
 
-export async function getAllShapesFromOrg(orgAuth: OrgAuthorization): Promise<OrgShapeListResult[]> {
+// it only really uses 3 properties from the Authorization type
+export async function getAllShapesFromOrg(
+  orgAuth: Pick<OrgAuthorization, 'orgId' | 'username'> & Partial<Pick<OrgAuthorization, 'aliases'>>
```

Comment:
@mshanemc I don't think it is necessary to constrain the type like this.

Reply:
yeah, this was done to limit the amount of stuff that has to be populated to be a full "OrgAuthorization" (limiting it to the 3 props this function needs).

See resulting change in the test.  Seems better?